src/pages/home/report/LinkPreviewer.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import {Image, View} from 'react-native';
+import Text from '@components/Text';
+import TextLink from '@components/TextLink';
+import useTheme from '@styles/themes/useTheme';
+import useStyleUtils from '@styles/useStyleUtils';
+import useThemeStyles from '@styles/useThemeStyles';
+import variables from '@styles/variables';
+
+const IMAGE_TYPES = ['jpg', 'jpeg', 'png'];
+const MAX_IMAGE_HEIGHT = 180;
+const MAX_IMAGE_WIDTH = 340;
+
+type ImageData = {
+    /** The height of the image. */
+    height: number;
+
+    /** The width of the image. */
+    width: number;
+
+    /** The URL of the image. */
+    url: string;
+
+    /** The type of the image. */
+    type?: string;
+};
+
+type LinkMetadata = {
```

Comment:
This type was added in [this PR](https://github.com/Expensify/App/pull/28730/files), move it to `src/types/onyx/ReportAction.ts` file and import it here